.github/workflows/release-please.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - dev
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
```

Inline with naming convention on extension, proper cased name here

```suggestion
name: Release packages
```